### ISSUE TITLE
Change documentation to refer to git

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8335]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8335|
|Type|enhancement|
|Created|2016-05-17 09:48:20Z|
|Branch|https://github.com/twisted/twisted/tree/git-docs-8335|

Change documentation which now refers to Subversion, to instead refer to git.

See: https://github.com/twisted/twisted/pull/62

<details><summary>Searchable metadata</summary>

```
trac-id__8335 8335
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__None None
branch__git_docs_8335 git-docs-8335
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1463478500668723 1463478500668723
changetime__1463530484630943 1463530484630943
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl set owner to @hawkowl|
|-|-|

Thanks rodrigc, I'm happy with this. Spun the builders, will merge when they are (undoubtedly) green :)

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl set status to `closed`|
|-|-|

In changeset 7593887c815e67a7fe5705b5be804e72a65a9a15
```
#!CommitTicketReference repository="" revision="7593887c815e67a7fe5705b5be804e72a65a9a15"
Merge git-docs-8335: Update documentation to mention Git instead of SVN

Author: rodrigc
Reviewer: hawkowl
Fixes: #8335
```
